DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: doAzureParallel
 Type: Package
 Title: doAzureParallel
-Version: 0.6.0
+Version: 0.6.2
```

Comment:
You can remove this,
We will make a new PR with changing the DESCRIPTION version. We will also tag that PR as the commit version

Reply:
rolled back the change